content/chainguard/libraries/java/build-configuration.md

```diff
@@ -0,0 +1,243 @@
+---
+title: "Build Configuration"
+linktitle: "Build Configuration"
+type: "article"
+description: "Configuring Chainguard Libraries for Java on your workstation"
+draft: false
+tags: ["Libraries", "Java"]
+menu:
+  docs:
+    parent: "java"
+weight: 053
+toc: true
+---
+
+The configuration for the use of Chainguard Libraries depends on your build
+tools, continuous integration, and continuous deployment setups
+
+At a high level adopting the use of Chainguard Libraries consists of the
+following steps:
+
+* Remove local caches on workstations and CI/CD pipelines.This step ensures that
+  any libraries that were already sourced from other repositories are requested
+  again and the version from Chainguard Libraries is used instead of other
+  binaries.
+* Change configuration to access Chainguard Libraries via your repository
+  manager after the changes from the [global
+  configuration](/chainguard/libraries/java/global-configuration) are
+  implemented.
+
+These changes must be performed on all workstations of individual developers and
+other engineers running relevant application builds. They must also be performed
+on any build server such as Jenkins, TeamCity, GitHub or other infrastructure
+that builds the applications or otherwise downloads and uses relevant libraries.
+
+## Cloudsmith
+
+Build configuration to retrieve artifacts from Cloudsmith requires you to
+authenticate. Use your username and password for Cloudsmith in your build tool
+configuration.
+
+## JFrog Artifactory
+
+Build configuration to retrieve artifacts from Artifactory typically requires
+you to authenticate and use the identity token in the configuration of your
+build tool:
+
+1. Log in as user with access to the configured virtual repository.
+1. Select **Edit Profile** from the drop down in the top right corner from your
+   user name.
+1. Press **Generate Identity Token**.
```

Comment:
Can you add a step about filling in the identity token description field with something meaningful, e.g. "Chainguard Java" or something like that